patroni/zookeeper.py

```diff
@@ -220,6 +220,11 @@ def delete_leader(self):
         if isinstance(self.leader, Leader) and self.leader.name == self._name:
             self.client.delete(self.client_path('/leader'))
 
+    def cancel_initialization(self):
+        node = self.get_node(self.initialize_key)
```

Comment:
get_node returns either tuple or None if the node doesn't exists
First element of tuple contains value of the node
Second - znode object